string RIGHT_HAND = "RIGHT_HAND"
string LEFT_HAND = "LEFT_HAND"


string touch_sensor_id